src/test/java/com/rudkids/rudkids/domain/user/domain/GenderTest.java

```diff
@@ -0,0 +1,23 @@
+package com.rudkids.rudkids.domain.user.domain;
+
+import com.rudkids.rudkids.domain.user.exception.InvalidGenderException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EmptySource;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+class GenderTest {
+
+    @DisplayName("성별 양식에 맞지 않을 시 예외 발생")
+    @ParameterizedTest
+    @ValueSource(strings = {"Male", "Female", " "})
```

Comment:
```suggestion
    @ValueSource(strings = {"Male", "Female", " ", ""})
```

이렇게 `빈 문자열`도 한 번 넣어봐요

Reply:
👍